agent_factory/integrations/telegram/admin/github_actions.py

```
@@ -0,0 +1,445 @@
+        try:
+            response = requests.post(url, json=data, headers=headers, timeout=10)
+
+            if response.status_code == 204:
+                logger.info(f"Triggered workflow: {workflow_filename}")
+                # Get latest run ID (GitHub doesn't return it in dispatch response)
+                return await self._get_latest_run_id()
+            else:
+                logger.error(f"Failed to trigger workflow: {response.status_code} - {response.text}")
+                return None
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)